packages/config/config/devices/0x0258/nas-pd02z-2.json

```
@@ -0,0 +1,261 @@
+					"value": 2
+				},
+				{
+					"label": "16 seconds",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Motion Sensor: Pulse Count",
+			"description": "Number of times motion must be detected within the measuring period (as defined by param 2: Motion Window Time) before a motion detected message is sent.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "1 time",
+					"value": 0
+				},
+				{
+					"label": "2 times",
+					"value": 1
+				},
+				{
+					"label": "3 times",
+					"value": 2
+				},
+				{
+					"label": "4 times",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Motion Sensor: Blind Time",
+			"description": "Time the sensor waits before sending a next motion detected message. This can be used to eliminate false positives by a light switching on or off due to the previously sent motion message.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"unit": "seconds",
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "0.5 seconds",
+					"value": 0
+				},
+				{
+					"label": "1 second",
+					"value": 1
+				},
+				{
+					"label": "1.5 seconds",
+					"value": 2
+				},
+				{
+					"label": "2 seconds",
+					"value": 3
+				},
+				{
+					"label": "2.5 seconds",
+					"value": 4
+				},
+				{
+					"label": "3 seconds",
+					"value": 5
+				},
+				{
+					"label": "3.5 seconds",
+					"value": 6
+				},
+				{
+					"label": "4 seconds",
+					"value": 7
+				},
+				{
+					"label": "4.5 seconds",
+					"value": 8
+				},
+				{
+					"label": "5 seconds",
+					"value": 9
+				},
+				{
+					"label": "5.5 seconds",
+					"value": 10
+				},
+				{
+					"label": "6 seconds",
+					"value": 11
+				},
+				{
+					"label": "6.5 seconds",
+					"value": 12
+				},
+				{
+					"label": "7 seconds",
+					"value": 13
+				},
+				{
+					"label": "7.5 seconds",
+					"value": 14
+				},
+				{
+					"label": "8 seconds",
+					"value": 15
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Motion Sensor",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "Motion Sensor: Clear Time",
+			"description": "Time in seconds to wait before sending a 'Clear motion detected' message after the last motion was detected. This determines the minimum time the sensor remains on after motion was detected. Possible values are 10 to 3600 seconds.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 3600,
+			"defaultValue": 30
+		},
+		{
+			"#": "7",
+			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
+		},
+		{
+			"#": "8",
+			"label": "Binary Sensor Report Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "9",
+			"label": "Motion Sensor: Basic Set Level",
```

```suggestion
			"label": "Motion Control: Basic Set Level",
```
to make a better correlation to the association group